Server/src/Server.Infrastructure/DomainServices/IdentityDomainService.cs

```diff
@@ -103,6 +103,25 @@ public async Task<AuthTokens> RefreshAccessTokenAsync(string oldRefreshToken, Ca
         return new AuthTokens(accessToken, refreshToken);
     }
 
+    public async Task LogoutAsync(string refreshToken, CancellationToken cancellationToken = default)
```

Comment:
Te metody w sumie nie powinny być w warstwie domeny bo te rzeczy nie dotyczą logiki biznesowej ale to tak tylko głośno myślę i nie ma co przenosić już